internal/cephfs/store/fsjournal.go

```diff
@@ -14,13 +14,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package core
+package store
```

Comment:
moving files to a new package should probably be a separate commit?

Reply:
as there is inter connection between a lot of things I kept it as a single commit.